<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormObjectLog</class>
 <widget class="QWidget" name="FormObjectLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ObjectLogWidget" name="objectLogWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelError">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Read DB fail&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEditFrom"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEditTo"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonMoveLeft">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonMoveRight">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonMoveNow">
       <property name="text">
        <string>Current</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTimePeriod">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>24 h</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionBackward">
   <property name="icon">
    <iconset resource="../Ui/Ui.qrc">
     <normaloff>:/Icons/Backward.png</normaloff>:/Icons/Backward.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous period</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../Ui/Ui.qrc">
     <normaloff>:/Icons/Forward.png</normaloff>:/Icons/Forward.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next period</string>
   </property>
  </action>
  <action name="actionCurrent">
   <property name="icon">
    <iconset resource="../Ui/Ui.qrc">
     <normaloff>:/Icons/DoNow.png</normaloff>:/Icons/DoNow.png</iconset>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
   <property name="toolTip">
    <string>Current period</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjectLogWidget</class>
   <extends>QWidget</extends>
   <header>ObjectLogWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Ui/Ui.qrc"/>
 </resources>
 <connections/>
</ui>
